nautobot/core/views/renderers.py

```diff
@@ -229,6 +231,21 @@ def get_context(self, data, accepted_media_type, renderer_context):
             "verbose_name_plural": queryset.model._meta.verbose_name_plural,
         }
         if view.action == "retrieve":
+            # Get the ObjectChange records to populate the advanced tab information
+            object_change_records = get_changes_for_model(instance)
```

Comment:
Since this is a duplicate, would it make sense to create a util function 
```python
created_by, last_updated_by = get_created_and_last_updated_usernames_for_model(instance)
```